Source/WebCore/platform/network/curl/CurlProxySettings.h

```diff
@@ -62,6 +62,10 @@ class CurlProxySettings {
     void setAuthMethod(long);
     long authMethod() const { return m_authMethod; }
 
+    using PlatformDataVariant = std::variant<Mode, std::pair<URL, String>>;
+    WEBCORE_EXPORT PlatformDataVariant platformSerializationData() const;
```

Comment:
I think the pattern in the rest of the code base is to call this `toIPCData()` and the variant `IPCData`.